packages/monitors/src/SyntheticPegMonitor.js

```diff
@@ -121,11 +139,11 @@ class SyntheticPegMonitor {
         message: "Synthetic off peg alert 😵",
         mrkdwn:
           "Synthetic token " +
-          this.empProps.syntheticCurrencySymbol +
```

Comment:
I renamed `syntheticCurrencySymbol` to `syntheticSymbol` throughout this model, the index, and other associated files for consistency.